### DISCUSSION TITLE
The $ prefix is reserved and cannot be used for variable and import namessvelte

### DISCUSSION THREAD

**Discussion by AxelBriche** [umaranis/svelte-lexical#41] [Q&A] [answered]
Hi, how have you solved this error inside the svelte-lexical core code please?
`The $ prefix is reserved, and cannot be used for variable and import namessvelte(illegal-declaration)`

**Comment by umaranis** [comment, marked as answer]
Hi @AxelBriche 

During import I provide an alias to the function which start with `$` symbol:

`import {$getSelection as getSelection} from 'lexical';`
